logux-vue-todomvc/babel.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
JSON-based `.babelrc` is much faster because it is easy to cache it (since it is not dynamic)

Reply:
It's generated by Vue CLI